webapp/src/pages/Settings.tsx

```diff
@@ -783,6 +793,69 @@ const Settings: React.FC<Props> = ({ onClose }) => {
     </FormGroup>
   );
 
+  const getCommonFieldsConfigSection = () => (
+    <Box marginY={1.5}>
+      <FormGroup>
+        <Columns columns={4}>
+          {displayReadonlyFields(
+            "artifact_path",
+            resultingConfig.artifact_path
+          )}
+          <NumberField
+            key="batch_size"
+            label="batch_size"
+            value={resultingConfig.batch_size}
+            disabled={isUpdatingConfig}
+            onChange={(newValue) =>
+              setPartialConfig({
+                ...partialConfig,
+                batch_size: newValue,
+              })
+            }
+            {...FIELDS["batch_size"]}
+          />
+          <StringField
+            select
+            label="use_cuda"
+            className="fixedWidthInput"
+            value={String(resultingConfig.use_cuda)}
+            disabled={isUpdatingConfig}
+            onChange={(newValue: string | boolean) =>
+              newValue &&
+              setPartialConfig({
+                ...partialConfig,
+                use_cuda: newValue === "auto" ? newValue : Boolean(newValue),
+              })
+            }
```

Comment:
There are two mistakes here that make it impossible to set `false`. Here is a suggestion that works:
```suggestion
            onChange={(newValue) => {
              const use_cuda =
                newValue === "true" || (newValue === "auto" && "auto");
              setPartialConfig({ ...partialConfig, use_cuda });
            }}
```

Reply:
Why did you leave the `newValue &&`? Won't it always be truthy?